docs/conf.py

```diff
@@ -89,7 +102,7 @@
 # The title in the left hand corner of the docs
 html_title = "Stravalib Docs"
 # Theme and css
-html_theme = "furo"
+html_theme = "pydata_sphinx_theme"
```

Comment:
This was just a temporary switch from furo to see if the theme rendered api docs differently or if it was in fact settings or my changes to the cod docs